core/src/com/unciv/ui/trade/DiplomacyScreen.kt

```diff
@@ -137,8 +137,8 @@ class DiplomacyScreen(val viewingCiv:CivilizationInfo):CameraStageBaseScreen() {
             friendBonusLabelColor = Color.GRAY
 
         val friendBonusLabel = friendBonusText.toLabel(friendBonusLabelColor)
-        diplomacyTable.add(friendBonusLabel).row()
-
+                .apply { setWrap(true); setAlignment(Align.center) }
+        diplomacyTable.add(friendBonusLabel).width(650f).row()
```

Comment:
Please use stage.width-dependant widths, so it will look good both on Android and on higher-resolution PC 🔢

Reply:
got it. i set the width to the one of `rightSideTable` minus some space of padding